bindata/v4.1.0/kube-controller-manager/pod.yaml

```diff
@@ -10,27 +10,17 @@ metadata:
     kube-controller-manager: "true"
     revision: "REVISION"
 spec:
-  initContainers:
-  - name: wait-for-host-ports
-    terminationMessagePolicy: FallbackToLogsOnError
-    image: ${IMAGE}
-    imagePullPolicy: IfNotPresent
-    command: ['/usr/bin/timeout', '65', '/bin/bash', '-ec'] # a bit more than 60s for TIME_WAIT, 5s extra cri-o's graceful termination period
-    args:
-    - |
-      echo -n "Waiting for port :10257 and :10357 to be released."
-      while [ -n "$(lsof -ni :10257)$(lsof -ni :10357)" ]; do
-        echo -n "."
-        sleep 1
-      done
   containers:
   - name: kube-controller-manager
     image: ${IMAGE}
     imagePullPolicy: IfNotPresent
     terminationMessagePolicy: FallbackToLogsOnError
-    command: ["/bin/bash", "-ec"]
+    command: ["/bin/bash", "-euxo", "pipefail", "-c"]
```

Comment:
with -x I suppose the echo statement are not needed

Reply:
this diverges from kube-apiserver operator. I don't like this.

Reply:
Might worth to fix it there too - `-u` and `-o pipefail` are good practice to prevent mistakes and typos.

`-x` is a matter of opinion - less pretty output but 10x easier to reason about when debugging

Reply:
I'm :+1: for `-x`, it can also be added to kas-o, as well.